server/graphql/loaders.js

```diff
@@ -258,7 +258,7 @@ export const loaders = req => {
           where: { ExpenseId: { [Op.in]: ExpenseIds } },
           group: ['ExpenseId'],
         })
-          .then(results => sortResults(ExpenseIds, results, 'ExpenseId'))
+          .then(results => sortResults(ExpenseIds, results, 'ExpenseId', { count: 0 }))
```

Comment:
Can you maybe land that patch as a separated PR to fast track it?

Reply:
Sure. I'll cherry pick this commit and remove ti from this PR.

Reply:
Done: https://github.com/opencollective/opencollective-api/pull/2877